### PR TITLE
Packaging

### PR DESCRIPTION
**Pull request by itaiMona**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+mona-sdk
+alt-profanity-check
```
Comment (TalzMona) [inline requirements.txt:2]: It looks like this library requires python 3.8 at least (while in our requirement file you request for 3.7 at least)

File: pyproject.toml [hunk 2]
```diff
@@ -1,2 +1,30 @@
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "mona-openai"
+version = "0.0.3"
```
Comment (TalzMona) [inline pyproject.toml:7]: Not really important - but maybe there's a way to delete the previous bad versions such that our first working version will be 0.0.1?

**Review by TalzMona — APPROVED** [review]
left 2 small comments. LGTM

File: pyproject.toml (hunk 2 above)
Comment (itaiMona) [reply pyproject.toml:7]: There isn't. Checked.

**Pull request merged**